### PR TITLE
Switchcell oncolor

### LINKED ISSUE #4027: Adding OnColor property for SwitchCell
## Summary
It would be nice to have the functionality to change the color of SwitchCell control since it's now available for the Switch control

## API Changes

```
public class SwitchCell : Cell
{
     public Color OnColor { get; set; }
} 
```
I would like to submit a PR if this proposal is accepted.

### PR DESCRIPTION
**Pull request by manutdkid77**
### Description of Change ###

Added OnColor property for changing color of SwitchCell on Android & iOS

### Issues Resolved ### 

- fixes #4027

### API Changes ###

Added:
 - Color OnColor { get; set; } //Bindable Property
 - UpdateOnColor() //Method to set/update the SwitchCell color

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android

### PR Checklist ###

- [ ] Has automated tests 
- [ ] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Comment by manutdkid77** [comment]
Not able to test the code on iOS as I don't have a Mac, will please need some help with this.

File: Xamarin.Forms.Core/Cells/SwitchCell.cs [hunk 1]
```diff
@@ -12,6 +12,14 @@ public class SwitchCell : Cell
 
 		public static readonly BindableProperty TextProperty = BindableProperty.Create("Text", typeof(string), typeof(SwitchCell), default(string));
 
+		public static readonly BindableProperty OnColorProperty = BindableProperty.Create("OnColor", typeof(Color), typeof(SwitchCell), Color.Default);
```
[the remark this answers is not in the document]
Comment (manutdkid77) [reply Xamarin.Forms.Core/Cells/SwitchCell.cs]: Sure, I will work on implementing this feature for UWP.

**Comment by samhouts** [comment]
Please also see the comments in #4049

**Comment by manutdkid77** [comment]
@samhouts have set the default Switch Color to use `UISwitch.Appearance.OnTintColor` from the Appearance API [Commit_Link](https://github.com/xamarin/Xamarin.Forms/blob/89bc195a39f781a65a6f6a158f490acfb1645fe9/Xamarin.Forms.Platform.iOS/Cells/SwitchCellRenderer.cs#L37)

**Comment by kingces95** [comment]
Awaiting UWP renderer...

**Comment by manutdkid77** [comment]
@kingces95 sorry for the late update, I wasn't able to do this for uwp. The [docs](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/custom-renderer/renderers#cells) mentioned a 
> DataTemplate with a Grid containing a TextBlock and ToggleSwitch

I didn't understand how to do this for uwp platform.

**Comment by paymicro** [comment]
UWP support https://github.com/manutdkid77/Xamarin.Forms/pull/1/commits/f63aea1308ae2b2bd9189785329b391769e947a5

**Pull request merged**